src/components/ImageView/ImageViewComponent.tsx

```diff
@@ -38,6 +39,11 @@ export const getImageCanvas = (padding: Padding, backgroundColor: string = "rgba
         regionCanvas = regionQuery[0] as HTMLCanvasElement;
     }
 
+    const catalogQuery = $(".catalog-plotly").children().children().children(".gl-container").children(".gl-canvas-context");
```

Comment:
should use `?` before each of the `.children()` values just in case

Reply:
Thanks